.postman/collection.json

```diff
@@ -867,7 +867,7 @@
 										{
 											"key": "mentor[photo]",
 											"type": "file",
-											"src": "./.postman/avatar.png"
+											"src": "/home/nelson/Pictures/sei.png"
```

Comment:
Can we upload a png file other than avatar.png?

Reply:
No, we should keep only images present in the repository. Will change that. Nice catch :raised_back_of_hand: